conans/cli/api/subapi/graph.py

```diff
@@ -83,11 +84,16 @@ def load_root_test_conanfile(self, path, tested_reference, profile_host, profile
         return root_node
 
     @api_method
-    def load_root_virtual_conanfile(self, ref, profile_host, is_build_require=False,
+    def load_root_virtual_conanfile(self, profile_host, requires=None, tool_requires=None,
```

Comment:
Would make sense a generic build_require instead the tool_requires?

Reply:
I am not sure, it tries to mimic the ``[tool_requires]`` we have in the ``conanfile.txt`` (and ``conanfile.py``). If we are adding a ``--build-requires``, then a explicit equivalent in conanfile should be created?
Actually it would make more sense to have a ``--test-requires``, the only thing is that for consumer ``conan install`` that is equivalent in practice to ``--requires``, so it is not strictly required functionality, just syntactic sugar.

Reply:
Agree